sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/ExpressionEvalHelper.scala

```diff
@@ -41,7 +41,12 @@ trait ExpressionEvalHelper extends GeneratorDrivenPropertyChecks {
   }
 
   protected def checkEvaluation(
-      expression: => Expression, expected: Any, inputRow: InternalRow = EmptyRow): Unit = {
+      originalExpr: => Expression, expected: Any, inputRow: InternalRow = EmptyRow): Unit = {
+    val expression = originalExpr match {
+      case replaceable: RuntimeReplaceable => replaceable.replaced
```

Comment:
Because when we add one that uses this it will fail if it doesn't work?

I can also remove this if you think it's better.